frappe/email/queue.py

```diff
@@ -348,7 +348,19 @@ def flush(from_test=False):
 				smtpserver = SMTPServer()
 				smtpserver_dict[email.sender] = smtpserver
 
-			send_one(email.name, smtpserver, auto_commit, from_test=from_test)
+			# send_one(email.name, smtpserver, auto_commit, from_test=from_test)
+			from frappe import enqueue
+			send_one_args = {
+				'email': email.name,
+				'smtpserver':smtpserver,
+				'auto_commit': auto_commit,
+				'from_test':from_test
```

Comment:
could you remove the comment above and fix the spacing in the entire dictionary?

Reply:
Yes @Thunderbottom

Reply:
@Thunderbottom Please check the changes and let me know if anything is required

Reply:
could you put the import to the start of the file instead?

Reply:
sure